py3status/modules/usbguard.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
+"""
+Usbguard i3 companion to replace default 'usbguard-applet-qt'.
+Show an indicator when a new usb device is plugged,
+we can allow or reject a device.
+
+Configuration parameters:
+    allow_urgent: Allow urgent (default False)
+    button_allow: Button to allow the device. (default 1)
+    button_block: Button to block the device (default None)
+    button_reject: Button to reject the device. (default 3)
+    format: Display format for the module. (default '[UsbGuard: [{device_name}]]')
+
+Format placeholders:
+    {device_name} the device name of last device plugged.
+    {device_id} the usbguard device id of last device plugged.
+    {device_hash} the usbguard device unique hash of last device plugged.
+
+Requires:
+    https://usbguard.github.io:
+    "The USBGuard software framework helps to protect your
+    computer against rogue USB devices..."
```

Comment:
(Nit/Opinion) Maybe just...
```
Requires:
    usbguard: usb device authorization policy framework 
```

Reply:
I have add deps. Ok, will study for change dbus lib. But not sure, hope it is not mandatory 🤔

Reply:
Not mandatory. A suggestion to kill a single dependency (or 5~ in my case). :wolf:

Reply:
Will try so.

Reply:
python dbus seems to make code unreadable...trying but really...legacy...